packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -554,24 +562,28 @@ const SUPPORTED_WIDGETS = [
     ...INDIVIDUAL_ANSWER_WIDGETS,
 ];
 
-export const getWidgetTypeFromWidgetKey = (widgetKey: string): string => {
-    return widgetKey.split(" ")[0];
-};
-
 /* Verify if the perseus item has supported widgets for automatic scoring */
-export const isWrongAnswerSupported = (widgetKeys: Array<string>): boolean => {
+export const isWrongAnswerSupported = (
+    widgetKeys: Array<string>,
```

Comment:
Question, nb: Were we trying to move away from using "keys" in relation to the widget name/ID? That's probably something to worry about another time, but just noticed "key" is used a lot here